src/main/java/me/zeroeightsix/kami/gui/rgui/Component.kt

```diff
@@ -20,21 +22,22 @@ open class Component(
 ) {
 
     // Basic info
-    val originalName = name
-    var name by setting("Name", name, { false })
-    protected val visibleSetting = setting("Visible", true, { false }, { _, it -> it || !closeable })
+    var nameKey : TranslationKey = if (nameT is TranslationKeyBlank) getTranslationKey("Name") else nameT
+    var name = setting(getTranslationKey("Name(Lit.)"), nameKey.value, { false })
```

Comment:
What is this supposed to mean?

Reply:
I have no idea what happened here, I believe that this is a change made by my python script that I have just gone along with whilst trying to fix all of the errors earlier. It isn't meant to mean anything.